cli/chaosli/cmd/create.go

```diff
@@ -0,0 +1,485 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/terminal"
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/types"
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "create a disruption.",
+	Long:  `creates a disruption given input from the user.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		spec, _ := createSpec()
+		jsonRep, err := json.MarshalIndent(spec, "", " ")
+		if err != nil {
+			fmt.Printf("json err: %v", err)
+		}
+
+		jsonRep = []byte(fmt.Sprintf(`{"apiVersion": "chaos.datadoghq.com/v1beta1", "kind": "Disruption", "metadata": %s, "spec": %s}`, getMetadata(), jsonRep))
+
+		y, err := yaml.JSONToYAML(jsonRep)
+		if err != nil {
+			fmt.Printf("yaml err: %v", err)
+		}
+
+		path, _ := cmd.Flags().GetString("path")
+		err = ioutil.WriteFile(path, y, 0644)
+		if err != nil {
+			fmt.Printf("writeFile err: %v", err)
+		}
+	},
+}
+
+const intro = `Hello! This tool will walk you through creating a disruption. Please reply to the prompts, and use ctrl+c to end.
+The generated disruption will have "dryRun:true" set for safety, which means you can safely apply it without injecting any failure.`
```

Comment:
Anyway we can bold the dryRun part. When I run the command it isnt automatically obvious that dryRun is turned on. Another alternative is giving them the option of turning dryRun off or on.

Reply:
I'm not sure it's straightforward to bold terminal output, so I'm going to switch to making this a flag that defaults to true